### DISCUSSION TITLE
Renovate autodiscover repo list only contains the pipeline repository (Azure DevOps)

### DISCUSSION THREAD

**Discussion by sowa705** [renovatebot/renovate#26679] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.137.1

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### What is your question?

I'm trying to refer to multiple repositories from a single pipeline but it seems that renovate can access only the repository its located in.

### Logs (if relevant)

<details><summary>Logs</summary>

```

 INFO: Autodiscovered repositories
       "length": 1,
       "repositories": ["project/renovaterepo"]
 WARN: Configured repository is in not in autodiscover list (repository=project/renovaterepo)
 INFO: Repository started (repository=project/renovaterepo)
       "renovateVersion": "37.137.1"
 INFO: Repository has no package files - skipping (repository=project/renovaterepo)
 INFO: Repository finished (repository=project/renovaterepo)
       "cloned": true,
       "durationMs": 2187

```

</details>

**Comment by rarkins** [comment]
Then you need to grant it more access and/or use a different token with the right access

**Reply by earvinnill** [reply]
do you have sample for this? would like to know as well.

**Reply by rarkins** [reply]
Create a user, its token, then add d that user as a developer to each repo you want it to run on.

**Reply by sowa705** [reply]
The build pipeline System.AccessToken is not enough?

**Reply by viceice** [reply]
PLease read the docs: https://docs.renovatebot.com/modules/platform/azure/#using-a-single-pipeline-to-update-multiple-repositories

**Reply by sowa705** [reply, marked as answer]
Actual answer: If you want to use System.AccessToken you need to change this setting in DevOps:
![image](https://github.com/renovatebot/renovate/assets/25490036/4b1f23b6-d13a-4862-9d14-0a4c7a2852d2)
If you don't want to change that you can use a different user
